app/Console/Kernel.php

```
@@ -30,6 +30,10 @@ protected function schedule(Schedule $schedule): void
         $schedule->command('mercator:certificate-expiracy')->daily();
         $schedule->command('mercator:cve-search')->daily();
         $schedule->command('cpe:sync')->dailyAt('00:30')->withoutOverlapping();
+
+        $schedule->call(function () {
+            app(LicenseService::class)->checkOnline();
```

PHPStan (2.1.32)

Call to method checkOnline() on an unknown class App\Console\LicenseService.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)